tests/unit/testsuite/Bolt/Boltpay/Controller/Traits/WebHookTraitTest.php

```diff
@@ -0,0 +1,441 @@
+<?php
+
+require_once('TestHelper.php');
```

Comment:
Please remove this. This is Magento plugin, not WordPress

Reply:
This is currently needed. 
Your request will be handled here: https://app.asana.com/0/544708310157130/1153588938387929